### ISSUE TITLE
Extra info activity

### ISSUE THREAD

**Issue by rayedgar** [bookingactivities/booking-activities#131]
If i do not bind each activity to an unique product, Is there a way to add additional info embedded somewhere in the activity settings that only shows this when selected on the calendar? So the Calendar view does not get cluttered?

**Comment by yoancutillas** [comment]
Yes, the [Display Pack](https://booking-activities.fr/en/downloads/display-pack/) add-on allow you to display additional info in a tooltip / below the calendar when you mouseover / click an event ([demo](https://demo.booking-activities.fr/en/shortcodes-integration/#booking-form), [docs](https://booking-activities.fr/en/docs/user-documentation/get-started-with-display-pack-add-on/set-additional-event-information-tooltip/)).
